### PR TITLE
1996 by Inlead: Implement the SPT project into DDBCMS.

### PR DESCRIPTION
**Pull request by inlead**
http://platform.dandigbib.org/issues/1996

### CODE REVIEW

File: modules/mkdru_ding/sass/mkdru_ding.scss [hunk 1]
```diff
@@ -13,10 +13,17 @@ a.cross:hover {
    ========================================================================== */
 
 .page-search-meta {
-
+  /* Hide header search form on the meta search page.
+  =========================================================================== git*/
```
Comment (guddo) [inline modules/mkdru_ding/sass/mkdru_ding.scss]: `git*`?

Comment (NicolaeGrigoriu) [reply modules/mkdru_ding/sass/mkdru_ding.scss]: Fiexd: https://github.com/ding2/ding2/pull/650/commits/cdda378ddb0e49772ab97c5f5bbfc08c857e2c95

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: 5472450dab9a1fa15d560b60a49933fbd123b8fb
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 54e3f95f6d4735179956aa52d2c66985fc82b6a6
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 1a17dee2e0df62c9492ef08da70a874262b16a84
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 84cb9581f3b6010bfd96c544694c0d28c6bb52eb
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: f9fe965e92575b89bec1747b23d3a28b95b143da
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 1c6f3637f88f8558e28f14189ecc12a7d597d0a4
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: ac810de21ba7c4d28ea13a8a2e943a5e524e105b
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 2dd589ca6a935e7dcaa7ed7540503e6033634cb8
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: a804a2acd7d0305188b17cb3abb66800b234ed83
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: f3a30cbd15ac327dd018f3c286338078c2dae3a2
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: ba416ede438e2f2b4fac30a2dc657a698e0d992b
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 055786fd8ea36b9aeed472502f623cae702b0023
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 59be92dc439b7304c827e3c3a7bb28690ee3f83e
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 1069f55c8a169bdb4381e6a6d6a92f847f7e9721
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: f76416047ffa14c4eeb2a9baa75a752eb015c4f1
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 838ce44beee826aabae344cffa718708e479afc6
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 6b676a151c8725d77523ae779de22c4e82144d4e
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 02095a4c94289ced74dcb2431c7b641ea073cc35
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: 5472450dab9a1fa15d560b60a49933fbd123b8fb
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 54e3f95f6d4735179956aa52d2c66985fc82b6a6
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 1a17dee2e0df62c9492ef08da70a874262b16a84
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 84cb9581f3b6010bfd96c544694c0d28c6bb52eb
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: f9fe965e92575b89bec1747b23d3a28b95b143da
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 1c6f3637f88f8558e28f14189ecc12a7d597d0a4
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: ac810de21ba7c4d28ea13a8a2e943a5e524e105b
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 2dd589ca6a935e7dcaa7ed7540503e6033634cb8
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: a804a2acd7d0305188b17cb3abb66800b234ed83
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: f3a30cbd15ac327dd018f3c286338078c2dae3a2
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: ba416ede438e2f2b4fac30a2dc657a698e0d992b
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 055786fd8ea36b9aeed472502f623cae702b0023
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 59be92dc439b7304c827e3c3a7bb28690ee3f83e
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 1069f55c8a169bdb4381e6a6d6a92f847f7e9721
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: f76416047ffa14c4eeb2a9baa75a752eb015c4f1
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 838ce44beee826aabae344cffa718708e479afc6
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 6b676a151c8725d77523ae779de22c4e82144d4e
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 02095a4c94289ced74dcb2431c7b641ea073cc35
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

**Comment by guddo** [comment]
Refactored and moved to https://github.com/ding2/ding2/pull/996.

**Pull request closed without merging**